docs/cdoc/qk-classical-register.rst

```diff
@@ -0,0 +1,37 @@
+=================
+QkClassicalRegister
+=================
```

Comment:
Docs will complain about these lines not being too long, it has to cover the whole word apparently:
```suggestion
===================
QkClassicalRegister
===================
```